sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -45,26 +41,23 @@ export class AttestationClient {
     constructor(credentials: TokenCredential, instanceUrl: string, options?: AttestationClientOptions);
     attestOpenEnclave(report: Uint8Array, options?: AttestOpenEnclaveOptions): Promise<AttestationResponse<AttestationResult>>;
     attestSgxEnclave(quote: Uint8Array, options?: AttestSgxEnclaveOptions): Promise<AttestationResponse<AttestationResult>>;
-    attestTpm(request: TpmAttestationRequest, options?: AttestTpmOptions): Promise<TpmAttestationResponse>;
+    attestTpm(request: Uint8Array, options?: AttestTpmOptions): Promise<Uint8Array | undefined>;
```

Comment:
Would it be a good idea to create a type alias for `Uint8Array` and use it here?

Reply:
An interesting question. This change came about because of feedback from the Python API review where they really didn't like the idea of a wrapper type. I'm not sure that a type alias would be any different in this case.

Reply:
a type alias is not a wrapper and has no runtime overhead and will slightly help the customer understand what to pass here. If the docstring for the method describes the parameter well and how to obtain it, I would not worry too much about it.

Reply:
Here's the docstring:

```
  /** Attest a TPM based enclave.

   * See the TPM Attestation Protocol Reference {@link https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol} for more information.
   * 
   * @param request - Incoming request to send to the TPM attestation service, Utf8 encoded.
   * @param options - Pipeline options for TPM attestation request.
   * @returns A structure containing the response from the TPM attestation, Utf8 encoded.
   */
```

The protocol reference is unfortunately the only documentation for this, it defines a JSON object which should be serialized and converted to UTF8.

Reply:
Oh interesting. Do you see opportunity to provide a better customer experience here in subsequent iterations?

btw I am fine with it as is for now.